src/main/java/org/alliancegenome/curation_api/model/entities/curationreports/CurationReportGroup.java

```diff
@@ -0,0 +1,24 @@
+package org.alliancegenome.curation_api.model.entities.curationreports;
+
+import javax.persistence.Entity;
+
+import org.alliancegenome.curation_api.base.entity.GeneratedAuditedObject;
+import org.alliancegenome.curation_api.view.View;
+import org.hibernate.envers.Audited;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Audited
+@Entity
+@Data @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString
+//@ToString(exclude = {"loads"})
+public class CurationReportGroup extends GeneratedAuditedObject {
+    
+    @JsonView({View.FieldsOnly.class})
+    private String name;
+}
```

Comment:
@markquintontulloch we will need the @OneToMany relationship here in order to get the reports page to have all the data

Reply:
@oblodgett - Done (I think!)

Reply:
@markquintontulloch the JsonView should probally be fields and lists and the `ToString` should exclude them. so that if you Log.info(curationReport) it doesn't print all the history. Sorry to keep adding.

Reply:
Will probably need create a custom view for this so we can control it better.

Reply:
Makes sense - done